### PR TITLE
pr_24

### PR DESCRIPTION
**Pull request by sns-seb**

### CODE REVIEW

**Review by sns-seb2 — DISMISSED** [review]
SonarQube has detected 3 issues

File: src/main/java/org/sonarqube/A.java [hunk 1]
```diff
@@ -3,7 +3,11 @@
 public class A {
   String message = "foo";
 
+  // TODO fix this
   public String foo() {
+    message = null;
```
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java]: ###AWArSQXM-3vUqknkNC1l###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_24@6
Complete the task associated to this TODO comment.

File: src/main/java/org/sonarqube/A.java [hunk 2]
```diff
@@ -3,7 +3,11 @@
...
+    message.length();
+
     return message;
```
Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java:11]: ###AWArSQXM-3vUqknkNC1m###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_24@9
The return value of "length" must be used.

Comment (sns-seb2) [inline src/main/java/org/sonarqube/A.java:11]: ###AWArSQXN-3vUqknkNC1n###
org.sonarqube:pr-decoration:src/main/java/org/sonarqube/A.java:BRANCH:pr_24@9
A "NullPointerException" could be thrown; "message" is nullable here.

**Review by sns-seb2 — DISMISSED** [review]
SonarQube has detected 2 issues

File: src/main/java/org/sonarqube/A.java (hunk 2 above)
Comment (sns-seb2) [reply src/main/java/org/sonarqube/A.java:11]: ===RESOLVED===
This issue has been resolved by a user in SonarQube

**Review by sns-seb2 — APPROVED** [review]

File: src/main/java/org/sonarqube/A.java (hunk 2 above)
Comment (sns-seb2) [reply src/main/java/org/sonarqube/A.java:11]: ===RESOLVED===
This issue has been resolved by a user in SonarQube

**Comment by sns-seb** [comment]
bug Bug

bug

the bug

bug

Bug Bugs :
vulnerability Vulnerabilities :
Code Smell Code Smells :

**Pull request closed without merging**